test/extension_tests/without_cmakelist_file/test/no_kits_present.test.ts

```diff
@@ -0,0 +1,46 @@
+import * as chai from 'chai';
+import * as chaiAsPromised from 'chai-as-promised';
+chai.use(chaiAsPromised);
+
+import {expect} from 'chai';
+
+import {clearExistingKitConfigurationFile, getExtension} from '../../../test_helpers';
+
+// This test will be skip when a Visual Studio installation marker (Env.HasVs=true) is present.
+// At the moment it is not possible to hide an installation against the test. In that case
+// it is not possible to test a no present kit, because VS will provid always kits.
+(process.env.HasVs == 'true' ? suite.skip : suite)('No present kit', () => {
+  let path_backup = '';
+  suiteSetup(() => {
+    clearExistingKitConfigurationFile();
+
+    // Test will use path to scan for compilers
+    // with no path content there is no compiler found
```

Comment:
with no path content no compiler will be found

Reply:
Yes. If there are no compilers then there is no kit present. What did you expect?

Reply:
I was commenting on the sentence structure.